common/scripts/emulators-ci

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+repo_root_dir=$(CDPATH= cd -- "$(dirname -- "$0")"/../.. && pwd)
+
+trap "kill 0" SIGINT
+
+(cd $repo_root_dir/packages/functions && rushx build ) &
+export CI=true && firebase emulators:start --project eisbuk $@
```

Comment:
I have a few reservation with this approach:

* there is no explanation in this commit about why this has been done; stating that it is "to fix ci tests" is not enough: why are they broken? Why should this fix them? What other possible solutions were evaluated and discarded?
* The `CI` variable [will already have a value](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) when run on github actions. So the utility of this script is to pretend to be in the CI when running locally. There is no explanation about why this would be a good idea.